torch/_subclasses/fake_tensor.py

```diff
@@ -1621,6 +1621,13 @@ def dispatch(self, func, types, args=(), kwargs=None):
         # e.g., manipulating args on constructor calls to construct meta tensors
         # and then afterwards wrapping them to a FakeTensor
         for run_impl_check, op_impl in op_implementations:
+            if func in (
+                aten._nested_tensor_from_tensor_list.default,
+                aten._nested_tensor_from_tensor_list.out,
+            ):
+                raise UnsupportedOperatorException(
+                    "torch.compile does not support C++ NestedTensor"
```

Comment:
for tracking: same nit as above - reference strided layout instead of C++ NestedTensor